src/Controls/src/Core/HandlerImpl/Ellipse.Impl.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui.Controls.Shapes2
```

Comment:
This namespace is temporary. Without using this namespace we would have many conflicts to resolve (mainly due to collision of types with Rectangle and Path), which would cause more than 200 files to be modified.
My proposal is, leave this PR with this temporal namespace to avoid so many changes, and apply the changes of namespaces in another specific PR.